Backend/item/urls.py

```diff
@@ -4,4 +4,5 @@
 
 urlpatterns = [
     path("", views.postItem),
+    path("get/<str:type>", views.getItemByType),
```

Comment:
The URL should not contain verbs, because the verb "get" is already present in the call method type.